### ISSUE TITLE
Measurement

### ISSUE THREAD

**Issue by karelstanek** [raugustyn/TB04CUZK001_Stanek#1]

**Comment by raugustyn** [comment]
What's up???

**Comment by karelstanek** [comment]
Snažím se nastavit obsah abychom tam mohli davat algoritmy, musim si neco o githubu přečíst :)

**Comment by raugustyn** [comment]
čtení nepomůže, je to strašlivě komplexní věc. to chce spíš rychlou indoktrinaci.

**Comment by karelstanek** [comment]
No potřebuju vedět jak přidat hochy aby mohli vkladat kod, kam psat ukoly, kam psat pravidla

**Comment by raugustyn** [comment]
K přidání dalších spolupracovníků:
1. Přidat řádek do [Datové a komunikační infrastruktury] (https://docs.google.com/spreadsheets/d/1UlJsMzt8gvjiZA5yCEPkv58avjv7spB4psakbYl2yOA/edit?usp=sharing), postačí tedy vyplnit jenom sloupeček GitHub
2. Vytvořit Issue v [Administrativě projektu](https://github.com/raugustyn/TB04CUZK001_WorkingGroup/milestones/Administrativa%20projektu) s jednoduchým popisem "přidejte mi pepíka a pavla do repositáře xxx" a Asignee nastavit na mne.

Zbytek udělám já, až bude hotovou napíšu tam message. Až prověříte že Vám to funguje, tak Issue uzavřete.

**Comment by raugustyn** [comment]
Úkoly jsou dvojího typu:
1. Úkoly programátorského charakteru - Issue a milestones v repository
2. Pravidla a diskuze k nim - od toho je repositář [KnowledgeDiscussions](https://github.com/raugustyn/TB04CUZK001_KnowledgeDiscussions).


Jinými slovy, ti vaši hoši by měli vidět i tam, aby viděli podrobné zadání.


oky???

**Comment by karelstanek** [comment]
ok, jen mi pošlou jejich id v githubu, jde issues nějak strukturovat (téma - operátor - algoritmus - varianty)?

**Comment by raugustyn** [comment]
Jsem udělal téma v otevřené diskuzi https://github.com/raugustyn/TB04CUZK001_KnowledgeDiscussions/issues/261

**Comment by KamilSvoboda** [comment]
Na poslední schůzce jsme to téma repository nedokončili, ale domníváme se že by bylo nejlepší začít pracovat od začátku nad jednou, společnou repository, která bude vnitřně strukturovaná dle jednotlivých "modulů". Než na konec pracně vše skládat dohromady.

Repository by se členila na adresáře podle jednotlivých metod přístupu a každý tým by si spravoval vlastní podadresáře. Bylo by tak snadné si vyzvednout celý obsah, dělat křížové vazby, apod. 

Zamýšleli jsme se i nad tou strukturou a přišlo nám vhodné na nejvyšší úrovni rozdělit adresáře na "data" a dva základní přístupy k řešení ("metoda 1" a "metoda 2", resp. "lopata" a "pinzeta"). Z pracovních důvodů jsme si ji v této podobě již založili u nás - viz. [T-MAPY/GenSMD](https://github.com/T-MAPY/GenSMD)

Otázka ale je, pod kterým GitHub účtem tuto pracovní repository založit. Nebylo by nejlepší pro tento účel založit "GitHub organizaci" a pojmenovat ji podle projektu? Nebo repository založit pod oficiálním účtem některé ze zainteresovaných organizací (např. VÚGTK, pokud takový existuje)?.

**Comment by raugustyn** [comment]
> Otázka ale je, pod kterým GitHub účtem tuto pracovní repository založit. Nebylo by nejlepší pro tento účel založit "GitHub organizaci" a pojmenovat ji podle projektu? Nebo repository založit pod oficiálním účtem některé ze zainteresovaných organizací (např. VÚGTK, pokud takový existuje)?.

Ano, to má logiku. Placený účet tedy mám zatím jen já, ale takovou věc jsme chtěli vymyslet společně na jednání, ke kterému prozatím nedošlo.

**Comment by raugustyn** [comment]
> Zamýšleli jsme se i nad tou strukturou a přišlo nám vhodné na nejvyšší úrovni rozdělit adresáře na "data" a dva základní přístupy k řešení ("metoda 1" a "metoda 2", resp. "lopata" a "pinzeta"). Z pracovních důvodů jsme si ji v této podobě již založili u nás - viz. T-MAPY/GenSMD

To není zas tak jasné. Máme dva výsledné software R1 a R2, ke kterým bychom měli systémově dospět. Máme několik týmů a jednotlivců, pomocí kterých to chceme vyrobit. Většina participantů nemá s verzováním a prací na jednom kódu zkušenost. Jedna věc je přidělit Droydovi soubor kterej si obsluhuje a nikdo mu na něj nesahá a druhá věc tohle.

Základ je nastíněn v [dokumentu] (https://drive.google.com/file/d/0BwKRDr9pxRhaM1dxTjRub3dINlE/view?usp=sharing) co sepsal Honza Šlegr, je to samozřejmě pouze základ k diskuzi. Ta by ale měla být na živo. S panem Langrem jsme se shodli že ho musíme zorganizovat do poloviny února.

**Comment by LabyrinthTechnology** [comment]
> S panem Langrem jsme se shodli že ho musíme zorganizovat do poloviny února.

Souhlasím s tím, že "co nejdříve", ale dopředu upozorňuji, že 6. - 13. 2. jsem na dovolené, takže by se mi líbilo, kdyby taková schůzka byla buď před (což je asi brzy) nebo až po.

**Comment by janlangr** [comment]
Za nás se střídame na janích prázdninách první 2 týdny v únoru, takže jako reálnou vidím schůzku v týdnu od 15.2.

**Comment by LabyrinthTechnology** [comment]
> Za nás se střídame na janích prázdninách první 2 týdny v únoru, takže jako reálnou vidím schůzku v týdnu od 15.2.

Za mne ok.

@raugustyn A co Ty, jak jsi na tom s prázdninami?

**Comment by raugustyn** [comment]
Naše prázdniny jsou 29.2.-4.3.

**Comment by LabyrinthTechnology** [comment]
> Naše prázdniny jsou 29.2.-4.3.

Takže se mi to jeví tak, že nekonfliktní jsou týdny 15. - 19. 2. a 22. - 26. 2. 2015.

Je ještě někdo, kdo by na schůzce neměl chybět (ať již z VUGTK nebo z UK) a mohl by tím pádem rozhodnout, který z výše uvedených termínů je lepší?

**Comment by PavelR2** [comment]
Proverili jsme moznost ukladani zdrojovych kodu a dalsich potrebnych dat (nyni SHP) pro PostgreSQL v GIT a mame hotovu implementaci nastroju pro ukladani zdrojovych kodu (tabulek, funkci, view) do textovych souboru, ktere je pak mozne verzovat. Zaroven mame implemntovanu i "obnovu" techto zdrojovych souboru zpet do DB a to i do prazdne.
Momentalne jsou vyzadovany tyto zakladni predpoklady:
OS Windows
PostgreSQL 9.5 w32
PostGIS 2.2.1 w32
Python 3.3 w32
GDAL CORE 1.11.3 (w32 nebo x64)
Aktualni zdrojove kody jsou k dispozici ve vyse uvedenem GIT repository:
https://github.com/T-MAPY/GenSMD/tree/develop
